app/scripts/modules/core/src/pipeline/config/stages/pipeline/PipelineStageExecutionDetails.tsx

```diff
@@ -13,6 +15,13 @@ export function PipelineStageExecutionDetails(props: IExecutionDetailsSectionPro
     current,
   } = props;
 
+  const { result: pipelineConfigs } = useLatestPromise<IPipeline[]>(() => {
+    application.pipelineConfigs.activate();
+    return application.pipelineConfigs.ready();
```

Comment:
we have to activate and wait for readiness of the data source here, because on standalone views this data source isn't already activated when the page loads.